xml/System.Buffers/IMemoryOwner`1.xml

```diff
@@ -18,9 +18,24 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of elements to store in memory.</typeparam>
+    <summary>Identifies the owner of a block of memory who is responsible for disposing of the underlying memory appropriately. </summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+The `IMemoryOwner<T>` interface is used to define the owner responsible for the lifetime management of a <xref:System.Memory%601> buffer. An instance of the `IMemoryOwner<T>` interface is returned by the <xref:System.Buffers.ArrayPool%601.Rent%2A?displayProperty=nameWithType> and <xef:System.Buffers.MemoryPool%601.Rent%2A?displayProperty=nameWithType> methods. 
+
+While a buffer can have multiple consumers, it can only have a single owner at any given time. The owner can:
+
+- Create the buffer either directly or by calling a factory method.
+
+- Transfer ownership to another consumer. In this case, the previous owner should no longer use the buffer.
+
+- Destroy the buffer when it is no longer in use.
+
+   ]]></format>
+    
+    </remarks>
```

Comment:
I assume we'll have a subsequent PR that adds the relevant safety rules related to `IMemoryOwner<T>`, e.g. it shouldn't be disposed of while a reference to its memory is handed out, which generally means it shouldn't have a finalizer, etc.?

Reply:
do you have a place where these safety rules are discussed @stephentoub?